grammars/less.cson

```diff
@@ -282,15 +282,15 @@
   }
   {
     'match': '\\b(a|abbr|acronym|address|applet|area|article|aside|audio|b|base|bdi|bdo|big|blockquote|body|br|button|canvas|caption|center|cite|code|col|colgroup|data|datalist|dd|del|details|dfn|dialog|div|dl|dt|em|embed|eventsource|fieldset|figure|figcaption|footer|form|frame|frameset|(h[1-6])|head|header|hgroup|hr|html|i|iframe|img|input|ins|kbd|keygen|label|legend|li|link|main|map|mark|math|menu|menuitem|meta|meter|nav|noframes|noscript|object|ol|optgroup|option|output|p|param|picture|pre|progress|q|rb|rp|rt|rtc|ruby|s|samp|script|section|select|small|source|span|strike|strong|style|sub|summary|sup|svg|table(?!-)|tbody|td|template|textarea|tfoot|th|thead|time|title|tr|track|tt|u|ul|var|video|wbr)\\b'
-    'name': 'keyword.control.html.elements'
+    'name': 'entity.name.tag.html.css'
```

Comment:
Also, do you think it's better to change the boundary assertions to `(?<!\\w-)` and `(?!\\w-)` so this won't mess up with the tokenization of, say, `article-title` (as the name of a custom element)?